sky/data/storage.py

```diff
@@ -42,6 +42,8 @@
 # R2 to be an option as preferred store type
 STORE_ENABLED_CLOUDS = [clouds.AWS(), clouds.GCP()]
 
+STORE_TYPE_TO_CLOUD_TYPE = {'s3': 'AWS', 'gcs': 'GCP', 'r2': 'Cloudflare'}
```

Comment:
Instead of adding this dict, can we use the `StoreType` enum when translating storetype to cloud?

Reply:
This dictionary is used to interact with what `get_enabled_clouds()` return, which is in a form of the cloud platforms' names, AWS, GCP, Cloudflare. So I would need a mapping between the object storage name and the cloud platform. But `StoreType` currently doesn't have a mapping to the cloud platform's name. What about adding something like `S3_CLOUD_TYPE = 'AWS'` under `StoreType` enum?

Reply:
Can we add a method like this to `StoreType`:
```
def to_cloud_str(self) -> str:
    if self == StoreType.S3:
        return str(clouds.AWS())
    elif self == StoreType.GCS:
        return str(clouds.GCP())
    elif self == StoreType.R2:
        return 'Cloudflare'
    else:
        with ux_utils.print_exception_no_traceback():
            raise ValueError(f'Unknown store type: {self}')
```
The goal is to minimize the number of mappings stored in different variables/constants. That way if we have to update this in the future, we know that only the `StoreType` requires updates.

Reply:
Removing this entirely as checking enalbed_storage_clouds in each Store classes' `_validate()` resolves the necessity.